.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We need to switch off babel module transpilation to allow webpack to use his own resolving module system: `"presets": [
    ["env", {
      "modules": false
    }]
  ]`

Reply:
✔️ Done